src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1987,3 +2017,139 @@
     text: >
         az tag delete --name MyTag
 """
+
+helps['template-specs'] = """
+type: group
+short-summary: Manage template specs at subscription or resource group scope.
+"""
+
+helps['template-specs create'] = """
+type: command
+short-summary: Creates or updates a template spec and or template spec version.
+parameters:
+  - name: --resource-group -g
+    type: string
+    short-summary: Name of resource group.
+  - name: --name -n
+    type: string
+    short-summary: Name of the template spec.
+  - name: --version -v
+    type: string
+    short-summary: The template spec version.
```

Comment:
May I ask is this the api-version for template spec?

Reply:
No it is not. There is a root template spec and versions of the root template-spec. The version is the name of the child template-spec.

Reply:
Got it, thanks~